codegen/module.go

```diff
@@ -83,6 +83,7 @@ type Options struct {
 	EnableCustomInitialisation bool
 	CommitChange               bool
 	QPSLevelsEnabled           bool
+	ProxyTemplates             *Template
```

Comment:
maybe a better name like custom templates to make it generic ? just a thought no hard opinions

Reply:
Done